retriever/database.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS `retriever_rule` (
```

Comment:
Addon's specific tables are now handled by the `dbstructure_definition` hook with a PHP array structure, see https://github.com/friendica/friendica-addons/blob/f573907bd210bbf19be9ad938d75394ba609ad70/advancedcontentfilter/advancedcontentfilter.php#L57-L97